.gitpod.yml

```diff
@@ -33,7 +33,6 @@ tasks:
     init: |
       leeway exec --package components/supervisor-api/java:lib --package components/gitpod-protocol/java:lib -- ./gradlew build
       leeway exec --package components/ide/jetbrains/backend-plugin:plugin -- ./gradlew buildPlugin
-      leeway exec --package components/ide/jetbrains/gateway-plugin:plugin -- ./gradlew buildPlugin
```

Comment:
it was renamed to `publish`, in workspace it will only build: https://github.com/gitpod-io/gitpod/blob/324b20088b5a7b6352732a80300e65c96151b4d0/components/ide/jetbrains/gateway-plugin/build.gradle.kts#L144

Reply:
already changed